src/styles/utils/_normalize.scss

```
@@ -0,0 +1,26 @@
+@use "./mixins" as *;
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font: inherit;
+}
+
+body {
+  min-height: 100vh;
+
+  @include flex;
```

Добавь какой-то комментарий-пояснение

```suggestion
  @include flex; // Will be removed later
```